{"name": "test_register[Create user - Mid Values]", "status": "failed", "statusDetails": {"message": "AssertionError: FAILURE: \u001b[31massert 200 == 201\ntests\\test_api.py:28 in test_register() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1", "trace": "FAILURE: \u001b[31massert 200 == 201\ntests\\test_api.py:28 in test_register() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1"}, "attachments": [{"name": "log", "source": "fab5eff6-8684-4e8d-90ad-0b35fc774516-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'Case': 'Create user - Mid Values', 'Test Type': 'Positive', 'Request_Body': {'firstName': 'JvlJVaIsnZulXdpmWUYiuzqIWE', 'lastName': 'WZgFSVdsCmKSyrEYsxKxERKgbM', 'username': 'fRgj5511', 'password': 'nY672'}}"}], "start": 1670168708097, "stop": 1670168708828, "uuid": "0f84b976-bb07-4aad-bcfe-d7187b05de5f", "historyId": "39586b0c56119deff7c77d22b62c41fb", "testCaseId": "bd51a8de3249e024de618e4e17a0a343", "fullName": "tests.test_api.TestApi#test_register", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-EGG762M"}, {"name": "thread", "value": "11972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}